### DISCUSSION TITLE
macOS(Montrey) won't run massCode

### DISCUSSION THREAD

**Discussion by UKONNsoft** [massCodeIO/massCode#66] [Q&A] [answered]
AS I install massCode  v2.5.0, I encountered the following dialog.
<img width="652" alt="スクリーンショット 2022-05-04 12 20 01" src="https://user-images.githubusercontent.com/7405040/166619346-ab13db4e-7210-4b30-992c-c94878bd70f0.png">
if you  force it to run, my mac reply with "this app is not right for current OS".
any  way to fix this?
macOS Montrey v12.3.1

**Comment by antonreshetov** [comment, marked as answer]
It seems that you have the Intel version of the mac and you are trying to install the ARM version.

**Reply by UKONNsoft** [reply]
Uohh....My mistake! excuse my carelessness..., thanks
